test/compliance/UPDATE.test.js

```diff
@@ -49,4 +50,93 @@ describe('UPDATE', () => {
       throw new Error('not supported')
     })
   })
+
+  describe('uniques in deep updates', () => {
+    cds.test(__dirname + '/resources')
```

Comment:
Please move the `describe` so that the file only has a single `cds.test`. It works for `cds-dbs`, but it does an additional deployment which should not be required for this test suite.